### DISCUSSION TITLE
What word am I?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#696] [Galaxy Trivia] [answered]
I am a word that can be used as a noun, a verb, an adjective, and an adverb.

**Comment by peterbaccaro** [comment, marked as answer]
The word that can be used as a noun, a verb, an adjective, and an adverb is "**run**".

As a noun, "run" refers to a period of time during which someone runs or a location for a race. For example: "I went for a run this morning."

As a verb, "run" means to move at a quick pace on foot or to operate a machine. For example: "I need to run to catch my flight" or "I need to run the dishwasher."

As an adjective, "run" describes something that is in progress or ongoing. For example: "I'm in the middle of a run of good luck."

As an adverb, "run" modifies a verb and means to move quickly or continuously. For example: "He ran quickly to catch the bus."

**Comment by nsafarova** [comment]
The word you are thinking of is "run."

As a noun, "run" refers to a period of time during which something operates or functions, or a route or course used for racing or traveling.

As a verb, "run" means to move quickly using your feet, or to operate or function a machine or system.

As an adjective, "run" can describe something that is in the process of operating or functioning, or something that has been extended or stretched out.

And finally, as an adverb, "run" can be used to describe the manner in which something is done quickly or continuously, as in "He ran quickly."
